### PR TITLE
[DEV APPROVED] 8959 - Styleguide component | Downloads box

### PR DESCRIPTION
**Pull request by johnplayer1982**
# 8959 - Styleguide component | Downloads box

[TP Ticket 8959](https://moneyadviceservice.tpondemand.com/entity/8959-styleguide-downloads-box)

Creates a new component in the fincap styleguide, specifications:

- Text colour: $color-white
- Background colour: $primary-green-pale
- Contains a heading (Always the same)
- Contains a list of downloads associated with the page content (So should be scalable)

image

### CODE REVIEW

**Comment by davidtrussler** [comment]
👍

File: app/views/styleguide/downloads.html.erb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      <li class="download-box__list-item">
+        <a href="http://www.fincap.org.uk" class="download-box__list-item-link">Download box content 1</a>
+      </li>
+      <li class="download-box__list-item">
+        <a href="http://www.fincap.org.uk" class="download-box__list-item-link">Download box content 2</a>
+      </li>
+      <li class="download-box__list-item">
+        <a href="http://www.fincap.org.uk" class="download-box__list-item-link">Download box content 3</a>
+      </li>
+    </ul>
```
Comment (margOnline) [inline app/views/styleguide/downloads.html.erb]: There is some repetitive markup in this file and in some of the other styleguide view files. Is it worth some simple drying up e.g.

```
<%- 3.time do |n| %>
  <li class="download-box__list-item">
    <a href="http://www.fincap.org.uk" class="download-box__list-item-link"><%= "Download box content #{n+1}"%></a>
  </li>
<% end %>
```

File: app/views/styleguide/downloads.html.erb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <ul class="download-box__list">
+      <li class="download-box__list-item">
+        <a href="http://www.fincap.org.uk" class="download-box__list-item-link">Download box content 1</a>
+      </li>
+      <li class="download-box__list-item">
+        <a href="http://www.fincap.org.uk" class="download-box__list-item-link">Download box content 2</a>
+      </li>
+      <li class="download-box__list-item">
+        <a href="http://www.fincap.org.uk" class="download-box__list-item-link">Download box content 3</a>
+      </li>
```
Comment (margOnline) [inline app/views/styleguide/downloads.html.erb]: Same comment as above.

File: app/views/styleguide/_menu.html.erb [hunk 3]
```diff
@@ -31,4 +31,7 @@
   <li class="styleguide-nav__item">
     <a href="/styleguide/latestnews" class="styleguide-nav__item-link">Latest News</a>
   </li>
+  <li class="styleguide-nav__item">
+    <a href="/styleguide/downloads" class="styleguide-nav__item-link">Download Box</a>
+  </li>
 </ul>
```
Comment (margOnline) [inline app/views/styleguide/_menu.html.erb]: Same comment as below.

File: app/views/styleguide/downloads.html.erb (hunk 2 above)
Comment (johnplayer1982) [reply app/views/styleguide/downloads.html.erb]: For the code example we should not include any ruby and should represent the desired markup (after any server side processing)

File: app/views/styleguide/_menu.html.erb (hunk 3 above)
Comment (johnplayer1982) [reply app/views/styleguide/_menu.html.erb]: Not sure how to do this in ruby, not a ruby developer

Comment (margOnline) [reply app/views/styleguide/_menu.html.erb]: I think of you as a jack of all trades. I can push something.

Comment (margOnline) [reply app/views/styleguide/_menu.html.erb]: @johnplayer1982 wdyt?

Comment (johnplayer1982) [reply app/views/styleguide/_menu.html.erb]: Your solution in the recent commit is excellent, much nicer

**Comment by margOnline** [comment]
👍

**Pull request merged**